src/main/java/org/assertj/core/api/ThrowableAssertAlternative.java

```diff
@@ -208,6 +208,29 @@ public ThrowableAssertAlternative(final T actual) {
     return this;
   }
 
+  /**
+   * Verifies that the message of the actual {@code Throwable} does not contain the given description
+   * <p>
+   * Examples:
+   * <pre><code class='java'>
+   *     //assertion will pass
+   *     assertThatExceptionOfType(Exception.class).isThrownBy(codeThrowing(new Exception("boom")))
+   *                                               .withMessageNotContaining("bam");
+   *     //assertion will fail
+   *     assertThatExceptionOfType(Exception.class).isThrownBy(codeThrowing(new Exception("boom")))
+   *                                               .withMessageNotContaining("boom");
+   * </code></pre>
```

Comment:
good example, just need to be formatted nicely as in `withMessageContaining` (generate the javadoc to check the result)